build.assets/Dockerfile-centos7-fips

```diff
@@ -59,17 +69,17 @@ ENV GOPATH="/go" \
 
 RUN git clone https://github.com/ninja-build/ninja.git && \
     cd ninja && \
-    git checkout v1.9.0 && \
+    git checkout v1.10.2 && \
     ./configure.py --bootstrap && \
     mv ninja /usr/bin
 
 RUN mkdir -p /opt && cd /opt && \
     git clone https://github.com/google/boringssl.git && \
     cd boringssl && \
-    git checkout ae223d6138807a13006342edfeef32e813246b39 && \
+    git checkout 853ca1ea1168dff08011e5d42d94609cc0ca2e27 && \
     mkdir build && \
     cd build && \
-    scl enable llvm-toolset-7.0 "cd /opt/boringssl/build && cmake3 -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DFIPS=1 -DCMAKE_BUILD_TYPE=Release -GNinja .. && ninja"
+    scl enable llvm-toolset-13.0 "cd /opt/boringssl/build && cmake3 -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DFIPS=1 -DCMAKE_BUILD_TYPE=Release -GNinja .. && ninja"
```

Comment:
Why 13? Don't we need clang 12?

Reply:
Still waiting your response on this one.

Reply:
Sorry, I fixed this locally (which is why I resolved this), but just didn't push my change back.